src/pages/Main/Main.js

```diff
@@ -1,17 +1,120 @@
 import React, { Component } from 'react';
+import CardLarge from './CardLarge/CardLarge';
+import Carousel from '../../components/Carousel/Carousel';
 import Nav from '../../components/Nav/Nav';
 import Footer from '../../components/Footer/Footer';
 import Card from '../../components/Card/Card';
+import CategoryImg from '../../components/CategoryImg/CategoryImg';
+import Slogan from './Slogan/Slogan';
 import './Main.scss';
 
 export default class Main extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      carouselImg: [],
+      sloganImg: [],
+      cardLarge: [],
+      index: 0,
+    };
+  }
+
+  componentDidMount() {
+    fetch('data/CarouselData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          carouselImg: data.results.product_events,
+          cardLarge: data.results.category_events,
+          sloganImg: data.results.slogan_events,
+        });
+      });
+  }
+
+  handleClickPrev() {
```

Comment:
```suggestion
  handleClickPrev = () => {
```
이벤트에 콜백 함수로 넘겼을 때 this가 동적으로 바뀌면서 문제가 될 수 있어서 class 컴포넌트에서는 가급적 화살표 함수로 써주는걸 권장드립니다

Reply:
수정완료했습니다. React 문서를 찾아봤는데 그렇다면 피드백 주신 사항이 바인드와 같은 원리라고 이해하도 무방할까요?

Reply:
@Junghoon-P 넵 .bind()를 써주면 되지만 굳이 쓸 이유가 없어서 화살표 함수를 써주는 걸 권장 드려요